tests/cairo_run_test.rs

```diff
@@ -552,6 +552,16 @@ fn cairo_run_dict_integration() {
     .expect("Couldn't run program");
 }
 
+#[test]
+fn cairo_run_secp_integration() {
+    cairo_run::cairo_run(
+        Path::new("cairo_programs/secp_integration_test.json"),
```

Comment:
To follow a standard, rename this program to `secp_integration_tests.cairo/.json`

Reply:
Resolved: https://github.com/lambdaclass/cleopatra_cairo/pull/370/commits/728d3b98b31e40551563bfa2c9c3ad7ffe3f12d2
https://github.com/lambdaclass/cleopatra_cairo/pull/370/commits/48cc98f2a02fd33f33c20fd18cda334c20cc3f77